README.md

```diff
@@ -19,8 +19,6 @@
 
 **Warning: The book and the documentation are missing content. Amethyst is undergoing a lot of changes at the moment.**
 
-This project is a *work in progress* and is very incomplete; pardon the dust!
-
 ## Goals
 
 * massively parallel architecture
```

Comment:
Should start with an uppercase to follow how the other bullet points of the book were made.